clients/src/main/java/org/apache/kafka/clients/consumer/AcknowledgeType.java

```diff
@@ -44,15 +44,11 @@ public String toString() {
 
 
     public static AcknowledgeType forId(byte id) {
-        switch (id) {
-            case 1:
-                return ACCEPT;
-            case 2:
-                return RELEASE;
-            case 3:
-                return REJECT;
-            default:
-                throw new IllegalArgumentException("Unknown acknowledge type id: " + id);
-        }
+        return switch (id) {
+            case 1 -> ACCEPT;
+            case 2 -> RELEASE;
+            case 3 -> REJECT;
+            default -> throw new IllegalArgumentException("Unknown acknowledge type id: " + id);
```

Comment:
Will it be compatible with release version of Java (11)?

Reply:
Nice point, fixed in new commit